libs/ui-components/src/components/Device/EditDeviceWizard/steps/ApplicationInlineForm.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { FieldArray } from 'formik';
+import { Button, FormGroup, Grid, Split, SplitItem } from '@patternfly/react-core';
+import MinusCircleIcon from '@patternfly/react-icons/dist/js/icons/minus-circle-icon';
+import PlusCircleIcon from '@patternfly/react-icons/dist/js/icons/plus-circle-icon';
+
+import { useTranslation } from '../../../../hooks/useTranslation';
+import CheckboxField from '../../../form/CheckboxField';
+import UploadField from '../../../form/UploadField';
+import TextField from '../../../form/TextField';
+import ExpandableFormSection from '../../../form/ExpandableFormSection';
+import { InlineAppForm } from '../../../../types/deviceSpec';
+
+// TODO - Celia to confirm
```

Comment:
Kept the TODOs as there will be small changes related to existing PRs which likely will be merged before this one.